### DISCUSSION TITLE
How to run an oci_tarball inside of an sh_binary?

### DISCUSSION THREAD

**Discussion by njlr** [bazel-contrib/rules_oci#667] [Q&A] [answered]
I would like to run an image target like a tool from an sh_binary.

Is there an example of this? 

Related: https://github.com/bazel-contrib/rules_oci/issues/103 (But note that question was about running a test)

**Comment by thesayyn** [comment, marked as answer]
Same this https://github.com/bazel-contrib/rules_oci/blob/eebda6568c91a1c437764ed665a7929aa04f294f/examples/assertion/BUILD.bazel#L344-L349 but use sh_binary instead.

**Reply by njlr** [reply]
My understanding is that will load the image, but not run it?

**Reply by thesayyn** [reply]
Once the image is loaded, you can run it with `docker run` like you would outside of Bazel.
